app_web/src/main.js

```diff
@@ -103,6 +103,16 @@ async function main()
         state.renderingParameters.morphing = morphingEnabled;
     });
 
+    uiModel.cameraValuesExport.subscribe( _ => {
+        let camera = state.userCamera;
+        if(state.cameraIndex !== undefined)
+        {
+            camera = state.gltf.cameras[state.cameraIndex];
+        }
+        const cameraDesc = camera.getDescription(state.gltf);
+        copyToClipboard(JSON.stringify(cameraDesc));
```

Comment:
we should implement this as a "map"  operator that returns a observable for the camera description